wrapping/python/openmeeg/_make_geometry.py

```diff
@@ -63,17 +64,10 @@ def make_geometry(meshes, interfaces, domains):
                 f"vertices and triangles). Got {type(mesh)}"
             )
 
-    indmaps = dict()
-    geom = Geometry()
-    for name, mesh in meshes.items():
-        indmaps[name] = geom.add_vertices(mesh[0])
-
-    # Create meshes
-
-    for name, mesh in meshes.items():
-        om_mesh = geom.add_mesh(name)
-        om_mesh.add_triangles(mesh[1], indmaps[name])
-        om_mesh.update(True)
+    # Create geometry from list-of-lists
+    meshes = [[name, mesh[0], mesh[1]] for name, mesh in meshes.items()]
+    geom = Geometry(meshes)
```

Comment:
... and then it gets used directly to instantiate the `geom` from list-of-list-of-ndarray...